frontend/agasthyarkoodam-trek/agasthya-trek.css

```
@@ -0,0 +1,700 @@
+    --agasthya-bg-alt: #11221a;
+    --agasthya-card-bg: rgba(17, 34, 26, 0.8);
+    --agasthya-card-border: rgba(16, 185, 129, 0.22);
+
+    --agasthya-text: #f0fdf4;
+    --agasthya-text-dim: #a7f3d0;
+    --agasthya-text-muted: #6ee7b7;
+
+    --agasthya-focus: #34d399;
+    --agasthya-shadow: 0 10px 30px -10px rgba(0, 0, 0, 0.6);
+
+    --font-heading: 'Playfair Display', Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)